HousingRepairsSchedulingApi/Services/Drs/DrsService.cs

```diff
@@ -133,7 +135,8 @@ public async Task<int> CreateOrder(string bookingReference, string sorCode, stri
             {
                 var errorMessage = createOrderResponse.@return.errorMsg;
 
-                LambdaLogger.Log(errorMessage);
+                _logger.LogError($"An error occurred while attempting to create an order with booking reference {bookingReference}: {errorMessage}");
```

Comment:
Not sure if this is required, the exception may automatically log the error in Cloudwatch

Reply:
Exception is below